inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/coroutine/MessageActionsCoroutine.kt

```diff
@@ -138,7 +139,7 @@ internal class MessageActionsCoroutine(
      */
     @VisibleForTesting
     @SuppressWarnings("CanBeNonNullable")
-    internal fun handleAction(onClickBehavior: OnClickBehavior?) {
+    internal fun handleAction(onClickBehavior: OnClickBehavior?, campaignId: String) {
```

Comment:
nit: `campaignId` can be set to default empty here, changes in UT will be minimized

Reply:
updated
please check